.circleci/config.yml

```diff
@@ -97,17 +97,17 @@ jobs:
           command: ./scripts/verify-artifact-contents
```

Comment:
while we are here, line 121:
Publishing should be the first step since if it fails, we should NOT proceed **with** opening backport PR/GitHub **release, etc**.

Reply:
Whoops... didn't notice, could you fix this on your open PR?

Reply:
fine